h/search/core.py

```diff
@@ -16,6 +16,19 @@
     'reply_ids',
     'aggregations'])
 
+DEFAULT_FILTERS = [
+    query.DeletedFilter,
+    query.GroupFilter,
+    query.UserFilter,
+    query.AnyMatcher,
+    query.TagsMatcher]
+
+DEFAULT_REQUEST_FILTERS = [
```

Comment:
There are currently two types of filters, ones that take a request obj as an arg and ones that don't hence the two different lists here.

Reply:
Having two separate lists seems a perfectly pragmatic approach to me. I would suggest adding a comment above each variable declaration to just clarify the difference.